server/routes/journeys/develop-an-initial-plan/identified-needs/tests.cy.ts

```diff
@@ -0,0 +1,91 @@
+import { v4 as uuidV4 } from 'uuid'
+import { checkAxeAccessibility } from '../../../../../integration_tests/support/accessibilityViolations'
+import { injectJourneyDataAndReload } from '../../../../../integration_tests/utils/e2eTestUtils'
+import { Plan } from '../../../../@types/express'
+
+context('test /record-decision/additional-information', () => {
+  const uuid = uuidV4()
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubGetPrisoner')
+    cy.task('stubGetPrisonerImage')
+    cy.task('stubComponents')
+    cy.task('stubCsipRecordSuccessPlanPending')
+  })
+
+  it('should test identified-needs with no needs', () => {
+    navigateToTestPage()
+    checkAxeAccessibility()
+
+    cy.url().should('to.match', /\/identified-needs$/)
+
+    validatePageContents(false)
+  })
+
+  it('should test identified-needs with needs', () => {
+    navigateToTestPage([
+      {
+        closedDate: '2024-08-01',
+        createdDate: '2024-08-01',
+        identifiedNeed: 'a need goes here',
+        intervention: 'some intervention',
+        progression: 'progression goes here',
+        responsiblePerson: 'test stafferson',
+        targetDate: '2024-08-01',
+      },
+    ])
+    checkAxeAccessibility()
+
+    cy.url().should('to.match', /\/identified-needs$/)
+
+    validatePageContents(true)
+  })
+
+  const navigateToTestPage = (identifiedNeeds: Plan['identifiedNeeds'] = []) => {
+    cy.signIn()
+    injectJourneyDataAndReload(uuid, {
+      prisoner: {
+        firstName: 'TestName',
+        lastName: 'User',
+        cellLocation: '',
+        prisonerNumber: '',
+        prisonId: '',
+      },
```

Comment:
this can come from `stubGetPrisoner` if you use `/csip-record/${uuid}/develop-an-initial-plan/start` as your starting point.

alternatively if you don't use the /start url and just inject it, `stubGetPrisoner` above can be removed